workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,20 +49,22 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
         {
-            BuildConfig();
+            Launch("Launch standalone client", "local worker launch UnityClient default", "spatial_*.log");
```

Comment:
think you got the log file names wrong. need to be the other way around `spatial_*.log` for launching a deployment and vice versa

Reply:
Oops, fixed them 👍